lib/fileio/copy.go

```diff
@@ -272,7 +280,7 @@ func (c copier) copyDirContents(src, dst, origDst string, uid, gid int, preserve
 }
 
 // copyDir copies the directory at src to dst.
```

Comment:
I think it might be better to maintain the original naming. The current naming seems to copy the child of src but actually src itself is the child.

Reply:
ok

Reply:
ok